### PR TITLE
chore: update stats util

### PR DESCRIPTION
**Pull request by gkatsev**
- Make quality levels look a bit nicer
- Add separate audio and video buffer output

### CODE REVIEW

File: src/media-segment-request.js [hunk 1]
```diff
@@ -269,9 +269,13 @@ const transmuxAndNotify = ({
       });
       trackInfoFn = null;
 
-      audioStartFn(probeResult.audioStart);
+      if (probeResult.hasAudio) {
```
Comment (brandonocasey) [inline src/media-segment-request.js]: This seems more like a fix/different pull request

**Comment by gkatsev** [comment]
@brandonocasey yeah, eventually, I'd like to combine the stats page and index into one thing but probably should be done separately.

File: src/media-segment-request.js (hunk 1 above)
Comment (gkatsev) [reply src/media-segment-request.js]: yeah, this and another change I have can/should go in another PR, I'll make sure to remove it when I'm fully done with the changes I want to make for this.

**Comment by brandonocasey** [comment]
I just meant link to it in the `index.html` at the root of this project. I removed it as part of another pull request.

**Comment by gkatsev** [comment]
Ah, I see. I think it's fine to leave it out for now since I'd like to combine them.

**Pull request merged**